deployment/chart/README.md

```diff
@@ -0,0 +1,101 @@
+# code-server
```

Comment:
Could you please wrap this file at column 80? I've got another PR that changes the rest of the documentation to be wrapped at column 80.

Reply:
Don't worry about the table or codeblocks, just the paragraphs.

Reply:
Done